c5g7/openmc/lattices.py

```diff
@@ -0,0 +1,153 @@
+                                             [u, u, u, u, u, u, u, u, u, u, u, u, u, u, u, u, u],
+                                             [u, u, u, u, u, u, u, u, u, u, u, u, u, u, u, u, u]]
+
+lattices['MOX Unrodded Assembly'] = openmc.RectLattice(lattice_id=103, name='MOX Unrodded Assembly')
+lattices['MOX Unrodded Assembly'].dimension = [17, 17]
+lattices['MOX Unrodded Assembly'].lower_left = [-10.71, -10.71]
+lattices['MOX Unrodded Assembly'].pitch = [1.26, 1.26]
+m = universes['MOX 4.3%']
+n = universes['MOX 7.0%']
+o = universes['MOX 8.7%']
+g = universes['Guide Tube']
+f = universes['Fission Chamber']
+lattices['MOX Unrodded Assembly'].universes = [[m, m, m, m, m, m, m, m, m, m, m, m, m, m, m, m, m],
+                                               [m, n, n, n, n, n, n, n, n, n, n, n, n, n, n, n, m],
+                                               [m, n, n, n, n, g, n, n, g, n, n, g, n, n, n, n, m],
+                                               [m, n, n, g, n, o, o, o, o, o, o, o, n, g, n, n, m],
+                                               [m, n, n, n, o, o, o, o, o, o, o, o, o, n, n, n, m],
+                                               [m, n, g, o, o, g, o, o, g, o, o, g, o, o, g, n, m],
+                                               [m, n, n, o, o, o, o, o, o, o, o, o, o, o, n, n, m],
+                                               [m, n, n, o, o, o, o, o, o, o, o, o, o, o, n, n, m],
+                                               [m, n, g, o, o, g, o, o, f, o, o, g, o, o, g, n, m],
+                                               [m, n, n, o, o, o, o, o, o, o, o, o, o, o, n, n, m],
+                                               [m, n, n, o, o, o, o, o, o, o, o, o, o, o, n, n, m],
+                                               [m, n, g, o, o, g, o, o, g, o, o, g, o, o, g, n, m],
+                                               [m, n, n, n, o, o, o, o, o, o, o, o, o, n, n, n, m],
+                                               [m, n, n, g, n, o, o, o, o, o, o, o, n, g, n, n, m],
+                                               [m, n, n, n, n, g, n, n, g, n, n, g, n, n, n, n, m],
+                                               [m, n, n, n, n, n, n, n, n, n, n, n, n, n, n, n, m],
+                                               [m, m, m, m, m, m, m, m, m, m, m, m, m, m, m, m, m]]
+
+lattices['MOX Rodded Assembly'] = openmc.RectLattice(lattice_id=104, name='MOX Rodded Assembly')
+lattices['MOX Rodded Assembly'].dimension = [17, 17]
+lattices['MOX Rodded Assembly'].lower_left = [-10.71, -10.71]
+lattices['MOX Rodded Assembly'].pitch = [1.26, 1.26]
+m = universes['MOX 4.3%']
+n = universes['MOX 7.0%']
+o = universes['MOX 8.7%']
+r = universes['Control Rod']
+f = universes['Fission Chamber']
+lattices['MOX Rodded Assembly'].universes = [[m, m, m, m, m, m, m, m, m, m, m, m, m, m, m, m, m],
+                                               [m, n, n, n, n, n, n, n, n, n, n, n, n, n, n, n, m],
```

Comment:
Can you remove the extra two spaces before rows 2+ in this lattice? This is probably an issue that I had in my inputs.